.prettierignore

```diff
@@ -0,0 +1,2 @@
+spec/*
+migrations/*
```

Comment:
As spec is generated, not sure what to do? Same with migrations

Reply:
Makes sense